odpf/optimus/runtime_service.proto

```diff
@@ -187,6 +187,11 @@ service RuntimeService {
             post: "/v1/project/{project_name}/job/{job_name}/replay"
```

Comment:
Can you also check if this endpoint is working as expected? Because I don't see the `body` field being used here.

Reply:
have checked this and it wasn't working, so have added body. did the same for replay dry run too